packages/side-nav/src/SideNavItem.tsx

```diff
@@ -0,0 +1,301 @@
+import React, { ReactNode, ElementType } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import {
+  AriaCurrentValue,
+  HTMLElementProps,
+  createDataProp,
+} from '@leafygreen-ui/lib';
+import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
+import { LEFT_RIGHT_OFFSET } from './styles';
+import omit from 'lodash/omit';
+
+const sideNavItemContainer = createDataProp('side-nav-item-container');
+
+// container styles
+
+const defaultStyle = css`
+  box-sizing: border-box;
+  position: relative;
+  width: 100%;
+  margin: unset;
+  padding: 10px ${LEFT_RIGHT_OFFSET}px 10px ${LEFT_RIGHT_OFFSET}px;
+  min-height: 0;
+  display: flex;
+  flex-direction: column;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  text-align: left;
+  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
+  transition: background-color 150ms ease-in-out;
```

Comment:
So I was trying to figure out why the scale transition doesn't work between states... We shouldn't care about background color on this element. The background color should be applied to the `:before` pseudo-class instead.